src/internal/ppsutil/util.go

```diff
@@ -269,39 +269,6 @@ func JobInput(pipelineInfo *pps.PipelineInfo, outputCommit *pfs.Commit) *pps.Inp
 	return jobInput
 }
 
-// PipelineReqFromInfo converts a PipelineInfo into a CreatePipelineRequest.
```

Comment:
This was only used in `pachctl list pipeline --spec`, so I'm removing it.

Reply:
Ooh, this is actually being used in #6406, so maybe leave it in.